File: tests/fixtures/pika/basic/consumer.py
"""pika consumer surface."""

import pika


def on_message(ch, method, properties, body):
    pass


def start_consumer():
    conn = pika.BlockingConnection()
    channel = conn.channel()
    channel.basic_consume(queue='order.created', on_message_callback=on_message)
    channel.basic_consume(queue='audit.write', on_message_callback=on_message)

File: tests/fixtures/amqplib/basic/consumer.ts
import amqp from 'amqplib';

export async function startConsumer() {
  const conn = await amqp.connect('amqp://localhost');
  const channel = await conn.createChannel();
  await channel.consume('order.created', () => {});
  await channel.consume('emails', () => {});
}
